types/Transform.js

```diff
@@ -11,6 +11,11 @@
 
 import type {Config, Path} from 'types/Config';
 
+export type TransformedSource = {|
+  content: string,
+  sourceMap: ?Object | string,
```

Comment:
is that the way typescript compiler returns it by default?
if not, should we rename it to babel's defaults?
i think it was `{code, map}` shape